optimizer_wrapper.rb

```diff
@@ -98,22 +99,26 @@ def self.wrapper_vrp(api_key, services, vrp, checksum, job_id = nil)
 
     if services_vrps.any?{ |sv| !sv[:service] }
       raise UnsupportedProblemError.new('Cannot apply any of the solver services', inapplicable_services)
-    elsif vrp.restitution_geometry && !vrp.points.all?{ |point| point[:location] }
-      raise DiscordantProblemError, 'Geometry is not available if locations are not defined'
+    elsif vrp.restitution_geometry.any? && !vrp.points.all?(&:location)
+      raise DiscordantProblemError.new('Geometry is not available if locations are not defined')
```

Comment:
Can't we move this to the check_consistency? It seems out of order here. We can also change the error type to an ArgumentError because we don't know if there is an issue with the problem.

Reply:
Sure, if we have that logic, you can do it like that.

I was under the impression that we return DiscordantProblemError when the optimisation problem itself has "disaggreeing/discordant" params (like, visits_number > schedule_range[end]-[begin], because the problem has a disagreement in it) but when the issue is due to a config like this then it is a misuse of an Argument  -- i.e., asking a geometry without locations. I might be wrong, in any case, there still is a disaccord/discordance so it is okay.